straxen/contexts.py

```diff
@@ -101,6 +101,18 @@ def xenonnt_online(output_folder='./strax_data',
     return st
 
 
+def xenon1t_led(**kwargs):
```

Comment:
Don't you want  `xenonnt_led` here?

Reply:
I think so! I think it is a typo

Reply:
Alright, I'll push it with bootstrax if that is ok